controller/invitation.go

```diff
@@ -0,0 +1,82 @@
+package controller
+
+import (
+	"github.com/fabric8-services/fabric8-auth/app"
+	"github.com/fabric8-services/fabric8-auth/authorization/invitation"
+	invitationService "github.com/fabric8-services/fabric8-auth/authorization/invitation/service"
+	"github.com/fabric8-services/fabric8-auth/errors"
+	"github.com/fabric8-services/fabric8-auth/jsonapi"
+	"github.com/fabric8-services/fabric8-auth/log"
+	"github.com/fabric8-services/fabric8-auth/login"
+	"github.com/goadesign/goa"
+	"github.com/satori/go.uuid"
+)
+
+// InvitationController implements the invitation resource.
+type InvitationController struct {
+	*goa.Controller
+	invService invitationService.InvitationService
+}
+
+// NewInvitationController creates a invitation controller.
+func NewInvitationController(service *goa.Service, invitationService invitationService.InvitationService) *InvitationController {
+	return &InvitationController{Controller: service.NewController("InvitationController"), invService: invitationService}
+}
+
+// Create runs the create action.
+func (c *InvitationController) CreateInvite(ctx *app.CreateInviteInvitationContext) error {
+	currentUser, err := login.ContextIdentity(ctx)
```

Comment:
Use login.ContextIdentityAndUserIfNotDeprovisioned()
and call it currentIdentity instead.

Reply:
I'm guessing you meant `LoadContextIdentityIfNotDeprovisioned()` - I've updated to use this function.